API_README.md

```diff
@@ -56,7 +56,6 @@ Finally, APIs which are integrated with the new `handleProcessingResult` method
     -   [Facility API](API_README.md#post-apifacility)
     -   [Practitioner API](API_README.md#get-apipractitioner)
```

Comment:
it looks like you reverted the prior commit on this PR. looking into it now.

Reply:
fixing it up now

Reply:
What if I didn't like that commit? Kidding.... I merged current and pushed current so, sorry, I guess:) You're still eagle eye!

Reply:
git, turned out was the 3 commits over last day or so :)